### PR TITLE
etcdctl: v3 as default README

### PR DESCRIPTION
**Pull request by gyuho**
For our upcoming v3 release.

### CODE REVIEW

**Comment by gyuho** [comment]
Check code base if anything is refering to old README.md, and scripts/build-binary is referring to it, but it doesn't matter anymore.

File: etcdctl/README.md [hunk 1]
```diff
@@ -1,341 +1,829 @@
 etcdctl
 ========
 
-`etcdctl` is a command line client for [etcd][etcd].
-It can be used in scripts or for administrators to explore an etcd cluster.
+TODO: merge into README.md.
```
Comment (heyitsanthony) [inline etcdctl/README.md]: delete this?

**Comment by gyuho** [comment]
PTAL. Added v2 link and notes about ETCDCTL_API.

Thanks.

File: etcdctl/README.md [hunk 2]
```diff
@@ -1,341 +1,830 @@
 etcdctl
 ========
 
-`etcdctl` is a command line client for [etcd][etcd].
```
Comment (heyitsanthony) [inline etcdctl/README.md]: keep this too?

**Comment by gyuho** [comment]
Just added top link for `etcd`. PTAL. Thanks.

**Comment by heyitsanthony** [comment]
lgtm, defer to @xiang90

File: etcdctl/README.md [hunk 3]
```diff
@@ -2,340 +2,831 @@ etcdctl
 ========
 
 `etcdctl` is a command line client for [etcd][etcd].
-It can be used in scripts or for administrators to explore an etcd cluster.
+Make sure to set environment variable `ETCDCTL_API=3`. For etcdctl v2, please check [READMEv2][READMEv2].
```
Comment (xiang90) [inline etcdctl/README.md]: For using etcdctl with API v2, please check

**Comment by xiang90** [comment]
LGTM

**Comment by gyuho** [comment]
thanks.

**Pull request merged**